samples/WebSocketSample/WebSocketSample.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\..\build\dependencies.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
Actually maybe since there are no dependencies, it's not needed. Make sure you can run this sample and it should be sufficient.